api/commands/validate.ts

```diff
@@ -0,0 +1,52 @@
+import * as cronParser from "cron-parser";
+import { dataform } from "@dataform/protos";
+
+const EMAIL_REGEX = /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i;
+
+function validateEmail(email: string) {
+  return EMAIL_REGEX.test(email);
+}
+
+export function validateSchedules(schedules: dataform.schedules.SchedulesJSON) {
+
+  const errors = [];
+
+  if (schedules.defaultNotification && schedules.defaultNotification.emails) {
+    schedules.defaultNotification.emails.forEach(email => {
+      if (!validateEmail(email)) {
+        errors.push(`${email} is not a valid email address.`);
```

Comment:
suggest surrounding these validated protobuf fields (`email` in this case) in quotes, so that the error looks nicer:

```
fo1w1r3 is not a valid email address.
```

vs 

```
"fo1w1r3" is not a valid email address.
```